### ISSUE TITLE
Update the "Now" line automatically

### ISSUE THREAD

**Issue by fernandocani** [richardtop/CalendarKit#206]
### New Issue Checklist

- [x] Use the latest version of CalendarKit
- [x] I've read the [Contribution Guidelines](https://github.com/richardtop/CalendarKit/blob/master/CONTRIBUTING.md)
- [x] I've searched for [existing GitHub issues](https://github.com/richardtop/CalendarKit/issues)

### Issue Description
The current hour line start correctly, but does not update automatically, only if i update the full view.
Didn't found also how to get the line to update only it.
![IMG_0017](https://user-images.githubusercontent.com/13990282/58727405-a1031400-83ba-11e9-985f-fcebc37e9590.PNG)

#### Result I am trying to achieve

[INSERT API OR UI MOCKUPS HERE]
The same behavior as the native calendar. The line represents the device time.

**Comment by richardtop** [comment]
Hi, @fernandocani thanks for the report, could you please tell, which CalendarKit version you're using? You can check this in `Podfile.lock`

**Comment by fernandocani** [comment]
Hi @richardtop,

```
PODS:
  - CalendarKit (0.7.0):
    - DateToolsSwift
    - Neon
  - DateToolsSwift (4.0.0)
  - Neon (0.4.0)

DEPENDENCIES:
  - CalendarKit (from `../`)

SPEC REPOS:
  https://github.com/cocoapods/specs.git:
    - DateToolsSwift
    - Neon

EXTERNAL SOURCES:
  CalendarKit:
    :path: "../"

SPEC CHECKSUMS:
  CalendarKit: 4c9353c4a692c3110132c7c7b0f5173fc48903b9
  DateToolsSwift: 875d97ff9e3a5d54abdd67a269b3f51c757b71ab
  Neon: 7fb5a9327e7e57911843f608e08f4893a74deaf9

PODFILE CHECKSUM: b76013ff0b66b8e61c214e583d26b4c3d93a550b

COCOAPODS: 1.6.1
```

**Comment by richardtop** [comment]
Thank you for the report. It's clearly a bug. I'll take a look and fix it. Until now, you can roll back to a previous version of the library, e.g. `0.6.0` and see if the problem is still reproducible.

**Comment by richardtop** [comment]
Please, take a look at the `Podfile.lock` of your project, not the one provisioned with the demo library. The `Podfile.lock` should be at the root of your project directory (i.e. your app).

**Comment by richardtop** [comment]
Fixed in [0.7.1](https://github.com/richardtop/CalendarKit/tree/0.7.1).
Please, let me know, if the issue still persists.

**Comment by fernandocani** [comment]
Hi, it works now! 
Thanks for the fix and the attention.
Have a great week o/
